### ISSUE TITLE
Suggestion about Tray Icon with Level

### ISSUE THREAD

**Issue by keijuns** [xanderfrangos/twinkle-tray#577]
Could you pleasd add, brightness level display on the tray icon?

Some MS software support display batter level or volume level.

Plase condiser it.

![image](https://user-images.githubusercontent.com/123173855/231017826-fd634402-bef0-4963-8d58-33bcc2f2b8fc.png)

**Comment by xanderfrangos** [comment]
In order to implement this, I'd need one PNG of every number from 0 to 100, at 5 different sizes (see the 1x-4x sizes [here](https://github.com/xanderfrangos/twinkle-tray/tree/master/src/assets/tray-icons/dark)). Light/dark mode colors may also be needed depending on the number color. If someone wants to make all of that, I could wire it up. But I don't really have any intention on making it myself. 😅

**Comment by xanderfrangos** [comment]
Nope, there unfortunately has to be one file for each full number.
